src/components/modals/ErrorModal.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from "react-bootstrap/esm/Button";
+import Modal from "react-bootstrap/esm/Modal";
+
+type Props = {
+    message: any;
+    show: boolean;
+    handleClose: () => void;
+};
+
+export const InfoModal: React.FC<Props> = ({ message, show, handleClose }) => {
```

Comment:
To jest to okienko z informacją pokazujące błąd jak coś nie pójdzie w trakcie usuwania.